modules/gapi/misc/python/package/gapi/init.py

```diff
@@ -0,0 +1,257 @@
+        # So just aggregate cv.GOpaqueT instead of inheritance
+        def __new__(cls, argtype):
+            return cv.GOpaqueT(argtype)
+
+        class Bool():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_BOOL)
+
+        class Int():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_INT)
+
+        class Double():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_DOUBLE)
+
+        class Float():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_FLOAT)
+
+        class String():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_STRING)
+
+        class Point():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_POINT)
+
+        class Point2f():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_POINT2F)
+
+        class Size():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_SIZE)
+
+        class Rect():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_RECT)
+
+        class Any():
+            def __new__(self):
+                return cv.GOpaqueT(cv.gapi.CV_ANY)
+
+    @register('cv2')
+    class GArray():
+        # NB: Inheritance from c++ class cause segfault.
+        # So just aggregate cv.GArrayT instead of inheritance
+        def __new__(cls, argtype):
+            return cv.GArrayT(argtype)
+
+        class Bool():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_BOOL)
+
+        class Int():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_INT)
+
+        class Double():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_DOUBLE)
+
+        class Float():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_FLOAT)
+
+        class String():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_STRING)
+
+        class Point():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_POINT)
+
+        class Point2f():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_POINT2F)
+
+        class Size():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_SIZE)
+
+        class Rect():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_RECT)
+
+        class Scalar():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_SCALAR)
+
+        class Mat():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_MAT)
+
+        class GMat():
+            def __new__(self):
+                return cv.GArrayT(cv.gapi.CV_GMAT)
+
+        class Any():
+            def __new__(self):
+                return cv.GArray(cv.gapi.CV_ANY)
+
+
+gapi.init = init
```

Comment:
Do we really need to export this `gapi.init()` function? Or it is a workaround and it should be eliminated.

Reply:
It's WA. Was impossible to add something into `cv2` package.

Reply:
removed WA